backend/utils.js

```diff
@@ -158,27 +159,25 @@ function deleteJSONFile(name, type, customPath = null) {
 }
 
 
-function recFindByExt(base,ext,files,result)
+async function recFindByExt(base,ext,files,result)
 {
-    files = files || fs.readdirSync(base)
+    files = files || (await fs.readdir(base))
     result = result || []
 
-    files.forEach(
-        function (file) {
-            var newbase = path.join(base,file)
-            if ( fs.statSync(newbase).isDirectory() )
-            {
-                result = recFindByExt(newbase,ext,fs.readdirSync(newbase),result)
-            }
-            else
+    for (const file in files) {
```

Comment:
This needs to be `for (const file of files)` (for in loop is for objects, for of loop is for iterables like arrays)